torch/autograd/gradcheck.py

```diff
@@ -234,7 +269,9 @@ def gradcheck(
     check_sparse_nnz: bool = False,
     nondet_tol: float = 0.0,
     check_undefined_grad: bool = True,
-    check_grad_dtypes: bool = False
+    check_grad_dtypes: bool = False,
+    check_forward: bool = False,
```

Comment:
nit: Every time I see `check_forward`, I get a little confused between the "forward pass" of a model and "forward-mode AD". Maybe it is better to make these flags more explicit? `check_forward_mode_ad`